server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,28 @@
+import { Pasta, Review } from "../../models/index.js";
+
+class ReviewSeeder {
+  static async seed() {
+    const reviewData = [
+      {
+        title: "spaghetti is gooood!",
+        review: "It goes so well with chicken parm",
+        rating: "5",
+        pastaId: (await Pasta.query().findOne({ name: "spaghetti" })).id,
+      },
+      {
+        title: "spaghetti with meatballs",
+        review: "Sweedish meatballs with spaghet!",
+        rating: "4",
+        pastaId: (await Pasta.query().findOne({ name: "spaghetti" })).id,
```

Comment:
Rather than making this query twice (once for each review), make the query before and save it to a variable!